### ISSUE TITLE
[Documentation] Adding Command Tutorial should not have quotes for command set paths

### ISSUE THREAD

**Issue by petriomelony** [evennia/evennia#2229]
#### Documentation issue
https://github.com/evennia/evennia/wiki/Adding-Command-Tutorial

The paths shown to command sets are enclosed in quotes, but they should not be.

#### Suggested change
Remove quotation marks from paths, ie:

`self.cmdset.add("mycmdset.MyCmdSet", permanent=True)`
`@py from typeclasses.objects import MyObject; [o.cmdset.add("mycmdset.MyCmdSet") for o in MyObject.objects.all()]`

should be

`self.cmdset.add(mycmdset.MyCmdSet, permanent=True)`
`@py from typeclasses.objects import MyObject; [o.cmdset.add(mycmdset.MyCmdSet) for o in MyObject.objects.all()]`

**Comment by Griatch** [comment]
No, this is correct. You can both give the python-path as a string and add the imported class - both works.
